src/main/cc/simif_f1.h

```diff
@@ -33,6 +33,7 @@ class simif_f1_t: public virtual simif_t
 //    int rc;
     int slot_id;
     int edma_fd;
+    int edma_read_fd;
```

Comment:
Shouldn't these be renamed to xdma_write_fd and xdma_read_fd?

Reply:
fixed